docs/source/admin/quick_howto/cachegroup_fallback_qht.rst

```diff
@@ -0,0 +1,65 @@
+..
+..
+.. _cachegroup-fallback-qht:
+
+******************************
+Configure CacheGroup Fallbacks
+******************************
+
+.. seealso:: :ref:`tp-configure-cache-groups`
+
+#. Go to 'Topology', click on Cache Groups, and click on your desired Cache Group or click the + button to create a new Cache Group.
+
+	.. figure:: cachegroup_fallback_qht/00.png
```

Comment:
The instructions make reference to a '+' button, but this screenshot cuts off said button.

I'm also not sold on showing production Cache Groups, imo it'd be better to use mock data (preferably with CDN-in-a-Box) to avoid any possible legal problems. Of course, I'm not a lawyer so maybe this is totally fine.

Reply:
done. and you are right, i removed the production Cache Groups